test/application/team-validator.js

```diff
@@ -48,15 +48,15 @@ describe('Team Validator', function () {
 	});
 
 	it('should validate Gen 2 IVs', function () {
-		let team = Dex.fastUnpackTeam('|raikou|||hiddenpowerwater||||14,28,26,,,|||');
+		let team = Dex.fastUnpackTeam('|raikou|||hiddenpowerwater||1||14,28,26,,,|||');
```

Comment:
A refactor should not make valid teams invalid. Backwards compatibility is important!